samples/tls/monitoring/README.md

```diff
@@ -8,25 +8,48 @@ All components have TLS enabled on ingress. Due to limitations in the
 underlying Helm charts, some components might not have TLS enabled in-cluster.
 See the **Limitations and Known Issues** section below for details.
 
-If you use this sample for HTTPS for ingress, the following secrets must be manually populated in the `monitoring` namespace (or `MON_NS` value) **BEFORE** you run any of the scripts in this repository:
+## Ingress
+
+If you use this sample for HTTPS for ingress, the following secrets must be
+manually populated in the `monitoring` namespace (or `MON_NS` value)
+**BEFORE** you deploy cluster monitoring.
 
 * kubernetes.io/tls secret - `prometheus-ingress-tls-secret`
 * kubernetes.io/tls secret - `alertmanager-ingress-tls-secret`
 * kubernetes.io/tls secret - `grafana-ingress-tls-secret`
 
-Generating these certificates is outside the scope of this example. However, you can use the
-process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq) in SAS Viya Administration documentation and specify the `monitoring` namespace.
+Generating these certificates is outside the scope of this example. However,
+you can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq)
+in SAS Viya Administration documentation and specify the `monitoring` namespace.
+
+## In-Cluster TLS
+
+The monitoring components use the following secrets for the TLS certificates
+service-to-service (east-west) traffic:
+
```

Comment:
I would revise to "The monitoring components use these secrets for the TLS certificates that handle interactions between services:"

Reply:
done